data/174/619/565/5/1746195655.geojson

```diff
@@ -0,0 +1,104 @@
+{
+  "id": 1746195655,
+  "type": "Feature",
+  "properties": {
+    "edtf:cessation":"uuuu",
+    "edtf:inception":"uuuu",
+    "geom:area":0.0,
+    "geom:area_square_m":0.0,
+    "geom:bbox":"-88.182276,30.697247,-88.182276,30.697247",
+    "geom:latitude":30.697247,
+    "geom:longitude":-88.182276,
+    "iso:country":"US",
+    "lbl:latitude":30.697247,
+    "lbl:longitude":-88.182276,
+    "lbl:max_zoom":18.0,
+    "mps:latitude":30.699142,
+    "mps:longitude":-88.18386,
+    "mz:hierarchy_label":1,
+    "mz:is_current":1,
+    "mz:max_zoom":18.0,
+    "mz:min_zoom":14.0,
+    "name:ara_x_preferred":[
+        "\u062c\u0627\u0645\u0639\u0629 \u062c\u0646\u0648\u0628 \u0623\u0644\u0627\u0628\u0627\u0645\u0627"
+    ],
+    "name:arz_x_preferred":[
+        "\u062c\u0627\u0645\u0639\u0629 \u062c\u0646\u0648\u0628 \u0627\u0644\u0627\u0628\u0627\u0645\u0627"
+    ],
+    "name:deu_x_preferred":[
+        "University of South Alabama"
+    ],
+    "name:eng_x_colloquial":[
+        "South"
+    ],
+    "name:eng_x_variant":[
+        "Usa"
```

Comment:
Perhaps we just ignore USA here altogether?

Reply:
fixed in cea7b0b